samples/SQL/allowed-sql-database-collation/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+   "properties": {
+     "displayName": "Allowed SQL Database collations",
+     "policyType": "Custom",
+     "description": "Will deny deploying databases with collations not listed as part of the allowed list",
+     "mode": "All",
+     "metadata": {
+       "category": "SQL"      
+     },
+     "parameters": {
+       "listOfAllowedCollations": {
+         "type": "Array",
+         "metadata": {
+           "displayName": "Allowed Collation Names",
+           "description": "A list of approved collations, example: Latin1_General_BIN;SQL_Latin1_General_CP1_CI_AS"
+         }
+       }
+     },
+     "policyRule": {
+       "if": {
+         "allOf": [
+           {
+             "field": "type",
+             "equals": "Microsoft.SQL/servers/databases"
+           },
+           {
+             "field": "name",
+             "notEquals": "master"
+           },
+           {
+             "not": {
+               "anyOf": [
+                 {
```

Comment:
in the interest of correctness you can get rid of this "anyOf" condition.  "anyOf" and "allOf" combine multiple conditions using OR or AND logical operators.  Since this only contains one condition ("field" "in") it isn't needed here.

Reply:
Thanks much !